packages/files/_test.pony

```diff
@@ -508,11 +469,19 @@ class iso _TestFileWritevLarge is UnitTest
       with file = CreateFile(filepath) as File do
         file.writev(wb.done())
       end
+
+      let offset: USize =
+        ifdef windows then
+          2
+        else
+          1
+        end
+
       with file2 = CreateFile(filepath) as File do
         count = 0
         while count < writev_batch_size do
-          let fileline1 = file2.line()?
-          h.assert_eq[String](count.string(), consume fileline1)
+          let fileline1 = file2.read_string(count.string().size() + offset)
+          h.assert_eq[String](count.string(), fileline1.split("\n")(0)?)
```

Comment:
Ugh! This won't work on Windows… it's `\r\n` not \n\r`. I'll fix these tests (again) later